test/canvas.test.js

```diff
@@ -447,29 +447,34 @@ describe('Canvas', function () {
   });
 
   it('Context2d#font=small-caps', function () {
+    if (process.platform == 'win32'){
+      // the windows version of pango (< 1.37.1) is too old to support font varients
```

Comment:
Doesn't have to be this PR (I can do it later unless you want to do it here) but it'd be better to expose the pango version (https://github.com/Automattic/node-canvas/blob/master/src/init.cc and https://github.com/Automattic/node-canvas/blob/master/index.js#L74) and test if it's older than 1.37.1. (Prebuilds use a newer version built from source.)

Reply:
Ah, good to know. I was wondering how to detect that directly...